### ISSUE TITLE
symbolic causes "'ascii' codec can't encode character..." error

### ISSUE THREAD

**Issue by profbrady** [Calysto/octave_kernel#23]
I am running octave_kernel in Jupyter and load the symbolic package. No problems creating symbolic variables. I can do addition and subtraction and get an answer. However, multiplication and division (and possibly more) cause errors.

multiplication:
` 'ascii' codec can't encode character u'\u22c5' in position 13: ordinal not in range(128)`

division:
` 'ascii' codec can't encode character u'\u2500' in position 22: ordinal not in range(128)`

Any solution? I am working on putting together materials for a class I teach on Octave to Engineering Technology students and think using Jupyter Notebooks would be great way to teach them the command line. I plan on uploading notebooks to our learning management system for students to download. The notebooks will have assignment instructions written with Markdown and Raw cells. However, I would like to include symbolic calculations for at least one class session.

Brian

**Comment by blink1073** [comment]
Hi @profbrady, what version of Python are you using, and can you post a sample of code that is causing the problem?

**Comment by profbrady** [comment]
I am using Anaconda with Python 2.7.10-2 on the Mac OS X El Capitan.


[Untitled5-Copy1.pdf](https://github.com/Calysto/octave_kernel/files/46116/Untitled5-Copy1.pdf)

**Comment by profbrady** [comment]
This is is not an issue on Windows 7 with Python 2.7.10. It does not work as all under Python 3.5 on Windows though. The symbolic package does not work with Python 3.5 for me under Anaconda, both in the Anaconda 3 command line and Jupyter.

**Comment by profbrady** [comment]
Well, now I feel foolish. I removed Anaconda 2 completely from my Mac and emptied the trash. Downloaded Anaconda 3 and installed it. Ran `conda update conda`, `pip install metakernel`, `pip install octave_kernel`, `pip install oct2py`, and `conda install jupyter` from terminal. Then I started the Anaconda launcher (Python 3.5.0_1) and a new Jupyter session. I opened the same notebook that gave the errors with symbolic calculations and everything worked great, no errors. I have not done anything to my Windows installation, so I still have symbolic problems with Python 3.5 on Windows not working at all, but it does work with Python 2.7.10.

Closing since it seems to work. I also have multiline output now and did not before.
